### DISCUSSION TITLE
Did pages visibility just become an enterprise feature rather than a team feature?

### DISCUSSION THREAD

**Discussion by jwillis0720** [community/community#43083] [Enterprise]
### Select Topic Area

Question

### Body

I just noticed our very private documentation via github pages just went public without warning. Before it was published to only members of the repository. 

The interface used to look like [this](https://github.blog/changelog/2021-01-21-access-control-for-github-pages/) but now it seems to be an enterprise feature? 

What am I missing? I don't see the ability to go private anymore unless I upgrade to enterprise.

**Comment by yoannchaudet** [comment]
Hello,

It has always been an enterprise feature. Feel free to reach support if you need any help understanding what could have happened.

**Reply by jwillis0720** [reply]
I still have 11 days left in my trial but its not working then. It's telling me to upgrade

**Reply by jwillis0720** [reply]
So even though I had 11 days. It for some reason is not working. Oh well. On to netlify
